internal/lookout/ui/src/containers/JobSetsContainer.tsx

```diff
@@ -229,6 +229,9 @@ class JobSetsContainer extends React.Component<JobSetsContainerProps, JobSetsCon
   }
 
   private async loadJobSets() {
+    if (this.state.queue === "") {
+      return
+    }
```

Comment:
This is a drive-by change that prevents us from making requests if no queue name is given.